setup.cfg

```diff
@@ -113,7 +113,7 @@ ignore = E741,W503,W504,H,E501,E203,D102
 # [D102] Missing docstring in public method
 # [D104] Missing docstring in public package
 # 88 is official black default:
-max-line-length = 88
-per-file-ignores =
+max_line_length = 88
+per_file_ignores =
```

Comment:
Really? https://black.readthedocs.io/en/stable/the_black_code_style.html does not indicate that.

Reply:
This change is coming from setuptools deprecation warning: https://setuptools.readthedocs.io/en/latest/history.html#v54-1-0

log during pep517 build
```
/tmp/pep517-build-env-ynrp5tfs/lib/python3.8/site-packages/setuptools/dist.py:634: UserWarning: Usage of dash-separated 'max-line-length' will not be supported in future versions. Please use the underscore name 'max_line_length' instead
/tmp/pep517-build-env-ynrp5tfs/lib/python3.8/site-packages/setuptools/dist.py:634: UserWarning: Usage of dash-separated 'per-file-ignores' will not be supported in future versions. Please use the underscore name 'per_file_ignores' instead
```